engine/table/build.gradle

```diff
@@ -85,10 +85,11 @@ dependencies {
     testImplementation platform(libs.junit.bom)
     testImplementation libs.assertj
 
-    testRuntimeOnly project(':log-to-slf4j')
     testRuntimeOnly project(':configs')
     testRuntimeOnly project(':test-configs')
-    testRuntimeOnly libs.slf4j.simple
+
+    runtimeOnly project(':log-to-slf4j')
```

Comment:
This change is to allow running the generator but probably shouldn't be in production. We need a logger because we use `QueryLanguageParser` which tries to load a logger.

Reply:
Is there any reason the generator needs to be in this module and not something standalone that doesn't make it onto the real classpath?